include/circt/Dialect/SV/SVStatements.td

```diff
@@ -437,6 +441,20 @@ def VerbatimOp : SVOp<"verbatim"> {
   ];
 }
 
+def VerbatimFileOp : SVOp<"verbatim.file"> {
+  let summary = "Verbatim opaque text emitted into a separate file.";
+  let description = [{
+    This operation produces opaque text in a separate SystemVerilog output file.
+    The `fileName` is a file path relative to other output where the `string`
+    will be written to.
+
+    Emitters honor a `nofilelist` unit attribute which if present prevent the
+    file from showing up in generated file lists.
+  }];
+  let arguments = (ins StrAttr:$fileName, StrAttr:$string);
```

Comment:
I don't really understand what is being modeled, here, but we can discuss this when it gets split out.